### PR TITLE
fix(filter-field): Focus is now placed on 'view more' button after ex…

### PR DESCRIPTION
**Pull request by Sherif-Elhefnawy**
After change:
quick-filter-group-focus

### CODE REVIEW

File: libs/barista-components/quick-filter/src/quick-filter-group.html [hunk 1]
```diff
@@ -50,7 +50,9 @@
       }"
     ></ng-container>
 
-    <button class="dt-show-more" (click)="_showMore()" i18n>View more</button>
+    <button #viewMoreButton class="dt-show-more" (click)="_showMore()" i18n>
+      <span>View more</span>
```
Comment (tomheller) [inline libs/barista-components/quick-filter/src/quick-filter-group.html:54]: Why is this span necessary in the button now? Should not make any difference.

**Review by tomheller — APPROVED** [review]
Can you check which `View More` gets focused, if you go into the details of the second group?

File: libs/barista-components/quick-filter/src/quick-filter-group.html (hunk 1 above)
Comment (Sherif-Elhefnawy) [reply libs/barista-components/quick-filter/src/quick-filter-group.html:54]: Due to some formatting shenanigans, space is added around 'View more', a span is added here to ensure this doesn't happen

**Comment by Sherif-Elhefnawy** [comment]
> Can you check which `View More` gets focused, if you go into the details of the second group?

Just for you, :D
quick-filter-group-focus-2

**Comment by Sherif-Elhefnawy** [comment]
Cherry-picked into:
10.8.x: 7fcfc918e6af74deaeff8c1fd34f5f3e7a85e062
a11y-fixes-barista-11: 73e1d64fcae47a6311ff87de927621662c4fb800

**Pull request merged**